plugins/google-sheets/src/pages/MapSheetFields.tsx

```diff
@@ -72,11 +72,17 @@ const inferFieldType = (cellValue: CellValue): CollectionFieldType => {
 
         if (/^#[0-9A-Fa-f]{6}$/.test(cellValue)) return "color"
 
-        if (/\.(jpeg|jpg|gif|png)$/.test(cellValue)) return "image"
+        if (/<[a-z][\s\S]*>/i.test(cellValue)) return "formattedText"
 
-        if (/^(https?:\/\/)?([a-zA-Z0-9.-]+)\.([a-zA-Z]{2,6})([/\w .-]*)*\/?$/.test(cellValue)) return "link"
+        try {
+            new URL(cellValue)
```

Comment:
If we're okay with Baseline 2023 features, then maybe [`URL.canParse`](https://developer.mozilla.org/en-US/docs/Web/API/URL/canParse_static)?

Reply:
Safari 16.6 doesn't support it, unfortunately.